### PR TITLE
[MISC]WOVN.php improvement

### PR DESCRIPTION
**Pull request by Soichiro725**
### Purpose/goal of PR (Issue #)
https://github.com/WOVNio/equalizer/issues/25125

- Delete `WOVNPHP_DEFAULT_INDEX_FILE`
- Add docker environment with Wordpress
- Add sample to not allow access `wovn.ini` to `htaccess_sample` 
- Add `.htaccess` to not allow access WOVN.php directly
- Reverse CircleCI tests order of PHP versions to find error version easily

### CODE REVIEW

File: htaccess_sample [hunk 1]
```diff
@@ -1,3 +1,8 @@
+# Don't allow access to wovn.ini direcly
```
Comment (bstewart00) [inline htaccess_sample]: typo `direcly` -> `directly`

**Comment by bstewart00** [comment]
This doesn't actually change anything right? The existing code had the `WOVNPHP_DEFAULT_INDEX_FILE` check. (Other than making it able to be a single string)

I wonder if this should be a wovn.ini setting instead?

**Comment by Soichiro725** [comment]
> This doesn't actually change anything right? The existing code had the `WOVNPHP_DEFAULT_INDEX_FILE` check. (Other than making it able to be a single string)
> 
> I wonder if this should be a wovn.ini setting instead?

I just allowed to set `WOVNPHP_DEFAULT_INDEX_FILE` with array.
But PHP 5.3 doesn't allow to set array with `define`.

I thought `wovn.ini` is good to set this config.
But there are some problems.
- Now `src/wovn_helper.php` doesn't have a dependency of Store, so I don't want to add dependency.
- It's better to not delete `WOVNPHP_DEFAULT_INDEX_FILE` to keep compatibility.

So, I'm thinking to add file name array option to `wovn_helper_detect_paths`.

File: .htaccess [hunk 2]
```diff
@@ -0,0 +1 @@
+Deny from all
```
Comment (a-bougouin) [inline .htaccess]: This new file is mistake?

Comment (Soichiro725) [reply .htaccess]: I wanted to block direct access to WOVN.php directory.

Comment (a-bougouin) [reply .htaccess]: Ah, I see

**Comment by bstewart00** [comment]
From the history it looked to me that the define variable was only added for test purposes, so I would question if there is really a backwards compatibility issue

**Comment by Soichiro725** [comment]
> From the history it looked to me that the define variable was only added for test purposes, so I would question if there is really a backwards compatibility issue

Yes. You are right.
Almost all users don't use WOVNPHP_DEFAULT_INDEX_FILE.
Even if user uses it, problem will happen when library is updated, but I think that updating is few times.
So I actually want to delete WOVNPHP_DEFAULT_INDEX_FILE.
Then, I will delete WOVNPHP_DEFAULT_INDEX_FILE.

**Comment by Soichiro725** [comment]
Finally, I just deleted `WOVNPHP_DEFAULT_INDEX_FILE`.
I will ask user to customize `wovn_helper.php` directly.
It is enough and simple.

**Pull request merged**